### PR TITLE
Vikram Lab 1

### PR DESCRIPTION
**Pull request by vbad23**
Just started. There is only an html and css file.

### CODE REVIEW

File: labs/lab-1-html-and-css/desktop-stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
+/*
+Author: Vikram Badarinath
+Created: 01/23/2018
```
Comment (diiq) [inline labs/lab-1-html-and-css/desktop-stylesheet.css:3]: Cute, but unnecessary, and in a real workplace, many people would edit this file, and you would always get the blame for their mistakes.

File: labs/lab-1-html-and-css/desktop-stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+*/
+
+body {
+  color: #69696a;
+  background-color: #f9fafc;
+  font-family: Open Sans;
+  font-weight: 800;
+}
+
+/* top of the page*/
```
Comment (diiq) [inline labs/lab-1-html-and-css/desktop-stylesheet.css:13]: Loving this commentary, though. And I love that you're thinking about organizing the file.

File: labs/lab-1-html-and-css/index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  </title>
+</head>
+
+<body>
+  <div class="menu">
+    <div class="title">
+      chirpus inc.
+    </div>
+    <div class="navbar">
+      <nav class="tab">
```
Comment (diiq) [inline labs/lab-1-html-and-css/index.html:24]: Ok, ok, some confusion here. `nav` is the element you should use for the *whole navbar*, not one element of the navigation.  So `<div class="navbar">` should be `<nav>` and `<nav class="tab"><a...` should be `<a class="tab">...`

File: labs/lab-1-html-and-css/index.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        <a href="contact.html">Contact</a>
+      </nav>
+      <nav class="special">
+        <a href="hire.html">Hire Us</a>
+      </nav>
+    </div>
+  </div>
+  <div class="content">
+    <div class="text">
+      <div class="bigTitle">
```
Comment (diiq) [inline labs/lab-1-html-and-css/index.html:43]: We use camelCase in javascript, but dash-case for class names in HTML.

File: labs/lab-1-html-and-css/index.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link href='http://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" href="desktop-stylesheet.css">
+  <link rel="stylesheet" href="stylesheet.css">
+  <!--
```
Comment (diiq) [inline labs/lab-1-html-and-css/index.html:9]: Ditto here. Your credit is in the *commit*, and doesn't belong in the file. Have you been looking at some old-school code somewhere? This is a very 1990's thing to do :)

File: labs/lab-1-html-and-css/desktop-stylesheet.css [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+  text-decoration: none;
+}
+
+.tab {
+  padding: 5px;
+  border: solid #f9fafc;
+  border-radius: 100px;
+}
+
+.special {
```
Comment (diiq) [inline labs/lab-1-html-and-css/desktop-stylesheet.css:47]: Nice noticing that you can share these styles between scroll down and hire us!

Maybe there's a better name for that class, though: `black-button` or something that you'll recognize and remember in a month.

**Review by diiq — COMMENTED** [review]
You killed it. Really excellent work.

File: labs/lab-1-html-and-css/index.html (hunk 5 above)
Comment (vbad23) [reply labs/lab-1-html-and-css/index.html:9]: I just noticed these comments, ha. They made us do it in our Java class so I thought that was some kind of standard. Good to know! Thanks for the feedback!

**Pull request closed without merging**